etl/steps/data/meadow/ons/2024-03-20/hours_and_earnings_uk.py

```diff
@@ -0,0 +1,114 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from typing import Dict, List
+
+from owid.catalog import Table
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+AGGREGATION_NAMES = {
+    "All employeesa": "All employees",
+    "Full-timea,b": "Full-time",
+    "Part-timea,c": "Part-time",
+}
+
+GROSS_WEEKLY_EARNINGS_CATEGORY_NAME = "Gross weekly earnings (£)"
+
+TEXTS_BELOW_TABLE = [
+    "a Employees on adult rates, whose pay for the survey period was unaffected by absence. Estimates for 2020 and 2021 include employees who have been furloughed under the Coronavirus Job Retention Scheme (CJRS).",
+    "b Full-time defined as employees working more than 30 paid hours per week (or 25 or more for the teaching professions).",
+    "c Part-time defined as employees working 30 paid hours or less per week (or less than 25 for the teaching professions).",
+    "Go to Contents",
+]
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot.
+    snap = paths.load_snapshot("hours_and_earnings_uk.xlsx")
+
+    # Load data from snapshot.
+    tb = snap.read(sheet_name="Table 5", header=2)
+
+    #
+    # Process data.
+
+    tb = reformat_table(tb, AGGREGATION_NAMES, GROSS_WEEKLY_EARNINGS_CATEGORY_NAME, TEXTS_BELOW_TABLE)
+
+    # Ensure all columns are snake-case, set an appropriate index, and sort conveniently.
+    tb = (
+        tb.underscore()
+        .set_index(["country", "year", "spell", "indicator", "aggregation"], verify_integrity=True)
+        .sort_index()
+    )
+
```

Comment:
you can now use the new Table method `format`:

```suggestion
    tb = tb.format(["country", "year", "spell", "indicator", "aggregation"])
```

Reply:
Thank you! I guess it was generated with a previous version of the wizard.

Reply:
ah, right, should update the wizard template!